ci/jenkins/test.sh

```diff
@@ -322,7 +322,7 @@ function prepare_env {
 function revert_snapshot_windows {
     WIN_NAME=$1
     echo "==== Reverting Windows VM ${WIN_NAME} ====="
-    govc snapshot.revert -vm ${WIN_NAME} win-initial
+    govc snapshot.revert -vm ${WIN_NAME} pristine-win-initial
```

Comment:
this function is also called when reverting windows jumper Node and it will revert it to snapshot `pristine-win-initial` which is wrong. Windows Jumper Node should be reverted to win-initial only and windows worker Node should be reverted to pristine snapshot. Please see https://github.com/antrea-io/antrea/pull/5052#issuecomment-1612897239